.jscsrc

```diff
@@ -1,5 +1,5 @@
 {
 	"preset": "jquery",
-
+	"maximumLineLength": 110,
```

Comment:
Reverted this change, found that the new version of jscs works - but grunt doesn't run it...

:frowning: -- such a pain to follow our style guide right now when the tooling isn't up to date.